### HYPOTHESIS TESTING`

#### Hypothesis

###### H1:The actual average weekly operating cost follows the theoretical cost model
###### H2:The actual average weekly operating cost is higher than the theoretical model predicts.

In [2]:
# -----------------------------------------------------------
# Hypothesis Testing: Weekly Operating Cost Model
# -----------------------------------------------------------

import math
from scipy.stats import norm

# Given data
sample_mean = 3050      # x̄
X = 600
theoretical_mean = 1000 + 5 * X   # μ = 1000 + 5X
sigma = 5 * 25          # population SD
n = 25                  # sample size
alpha = 0.05            # significance level

# Calculate test statistic (Z)
Z = (sample_mean - theoretical_mean) / (sigma / math.sqrt(n))

# Critical value (right-tailed test)
Z_critical = norm.ppf(1 - alpha)

print("Hypothesis Testing for Weekly Cost Model \n")
print(f"Theoretical Mean (μ) = {theoretical_mean}")
print(f"Sample Mean (x̄) = {sample_mean}")
print(f"Standard Deviation (σ) = {sigma}")
print(f"Sample Size (n) = {n}")
print(f"Z-statistic = {Z:.2f}")
print(f"Critical Z-value (α=0.05) = {Z_critical:.3f}\n")

# Decision
if Z > Z_critical:
    print("Reject Null Hypothesis (H₀)")
    print("Evidence supports that weekly costs are higher than the model suggests.\n")
else:
    print("Fail to Reject Null Hypothesis (H₀)")
    print("No evidence that weekly costs are higher than the model suggests.\n")


Hypothesis Testing for Weekly Cost Model 

Theoretical Mean (μ) = 4000
Sample Mean (x̄) = 3050
Standard Deviation (σ) = 125
Sample Size (n) = 25
Z-statistic = -38.00
Critical Z-value (α=0.05) = 1.645

Fail to Reject Null Hypothesis (H₀)
No evidence that weekly costs are higher than the model suggests.



#### Conclusion

##### The calculated Z-statistic is –38, while the critical Z-value for a right-tailed test at a 5% significance level is 1.645. Since the calculated Z (–38) is less than the critical value (1.645), we fail to reject the null hypothesis (H₀). This means there is no statistical evidence to support the restaurant owners’ claim that the weekly operating costs are higher than predicted by the cost model. The difference observed in sample mean (₹3050) compared to the theoretical mean (₹4000) could be due to random variation rather than a real increase. Therefore, the existing cost model still accurately represents the weekly operating cost structure.